### PR TITLE
Install from manifest

### PR DESCRIPTION
**Pull request by alexeymorozov**
See #15

### CODE REVIEW

File: lib/fontist/cli.rb [hunk 1]
```diff
@@ -64,9 +64,26 @@ def update
...
+      print_yaml(paths)
+      success
+    rescue Fontist::Errors::ManifestCouldNotBeFoundError
+      error("Manifest could not be found.")
+    rescue Fontist::Errors::ManifestCouldNotBeReadError
+      error("Manifest could not be read.")
+    end
+
+    desc "manifest-install MANIFEST", "Install fonts from MANIFEST (yaml)"
+    option :confirm_license, banner: "yes", desc: "Confirm license agreement"
```
Comment (alexeymorozov) [inline lib/fontist/cli.rb:80]: Will it be ok for us to have such an option? Could be used as:

```
fontist manifest-install --confirm-license=yes manifest.yml
```

Comment (ronaldtse) [reply lib/fontist/cli.rb:80]: Certainly, that works. Or can we do `fontist manifest-install --confirm-license manifest.yml`?

**Review by ronaldtse — APPROVED** [review]
LGTM! Feel free to merge when the comment is addressed. Thanks!

File: lib/fontist/cli.rb (hunk 1 above)
Comment (ronaldtse) [reply lib/fontist/cli.rb:80]: Ping @alexeymorozov

Comment (alexeymorozov) [reply lib/fontist/cli.rb:80]: Yeap, on it :) Thanks

**Pull request merged**